### PR TITLE
Release v5.5.1

### LINKED ISSUE #6659: Release 5.5.1

### PR DESCRIPTION
**Pull request by sellout**
Fixes #6659.

### CODE REVIEW

**Comment by nuttycom** [comment]
Compare link for changes since v5.5.0: https://github.com/zcash/zcash/compare/v5.5.0..a9658cc

**Review by str4d — APPROVED** [review]
utACK

**Review by nuttycom — APPROVED** [review]
utACK

File: doc/release-notes/release-notes-5.5.1.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+-----
+
+Fixes an issue that could cause a node to crash if the privacy policy does not
+include `AllowRevealedRecipients` when attempting to create a transaction that
+results in transparent change. See
+[#6662](https://github.com/zcash/zcash/pull/6662) for details.
+
+Also corrects an underestimate of the [ZIP 317](https://zips.z.cash/zip-0317)
+conventional fee when spending UTXOs, which can result in mining of the
+transaction being delayed or blocked. See
```
Comment (daira) [inline doc/release-notes/release-notes-5.5.1.md:14]:
```suggestion
conventional fee when spending transparent inputs, which can result in mining
of the transaction being delayed or blocked. See
```

**Review by daira — APPROVED** [review]
utACK with non-blocking comment.

**Pull request merged**